lib/restAPIs/userManagement.js

```diff
@@ -4,13 +4,12 @@ const assertParam = require('../util/assertParam');
 const RestUtil = require('../util/RestUtil');
 const buildCommonQuery = require('../util/buildCommonQueryParams');
 
-
 module.exports = class RestTopic extends RestUtil {
-  _getBase(){
+  _getBase() {
     return 'users';
   }
 
-  getUsers(nameBegins, abbreviationBegins, offset = 0, limit = -1, fields){
+  getUsers(nameBegins, abbreviationBegins, offset = 0, limit = -1, fields) {
```

Comment:
Since you're modifying userManagement.js, could you add JSDocs to the public methods?

Reply:
Done.